site/content/docs/developing/additional-aws-resources.ja.md

```diff
@@ -117,3 +117,36 @@ Outputs:
     
     * `addons/` ディレクトリで定義するポリシーでは[最小限のアクセス権を付与する](https://docs.aws.amazon.com/ja_jp/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
     * `addons/` ディレクトリで定義したリソースに対してのみアクセスできるようにポリシーを制限するために [追加セキュリティに対するポリシー条件を使用する](https://docs.aws.amazon.com/ja_jp/IAM/latest/UserGuide/best-practices.html#use-policy-conditions) 
+
+
+### `Parameters` セクションのカスタマイズ
+
+Copilot では、 `App`, `Env` そして `Name` [パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)がテンプレートに定義されている必要があります。もし、 Service スタック内のリソースを参照する[パラメーター](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)を追加したい場合は、 `addons.parameters.yml` ファイルを作成します。
```

Comment:
Can we update this sentence like this? (I think I understand your **awesome** intent here, but let me know if I'm not ‪🤗‬)

```suggestion
Copilot では、 `App`, `Env` そして `Name` パラメーターがテンプレートに定義されている必要があります。もし、 Service スタック内のリソースを参照するを追加したい場合は、 `addons.parameters.yml` ファイルを作成します。テンプレート内パラメーターのデータ型やデフォルト値の指定方法については、[CloudFormation テンプレートでのパラメーター定義](https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)に準じます。
```

Reply:
@toricls 
Thanks for the suggestion!  I don't think there is any particular difference in perception.
It is easier for me to understand what you are suggesting.